app/concerns/liquid_interpolatable.rb

```diff
@@ -221,6 +221,43 @@ def logger
           Logger.new(STDERR)
         end
     end
+
+    BACKSLASH = "\\".freeze
+
+    UNESCAPE = {
+      "a" => "\a",
+      "e" => "\e",
+      "f" => "\f",
+      "n" => "\n",
+      "r" => "\r",
+      "t" => "\t",
+      "v" => "\v",
+      BACKSLASH => BACKSLASH + BACKSLASH,
+      "1" => BACKSLASH + "1",
+      "2" => BACKSLASH + "2",
+      "3" => BACKSLASH + "3",
+      "4" => BACKSLASH + "4",
+      "5" => BACKSLASH + "5",
+      "6" => BACKSLASH + "6",
+      "7" => BACKSLASH + "7",
+      "8" => BACKSLASH + "8",
+      "9" => BACKSLASH + "9",
+    }
+
+    def unescape_replacement(s)
```

Comment:
if possible I would also like to understand this code. Some interesting stuff going on here (to a ruby noob).